src/js/user-controller/user-controller-events.js

```diff
@@ -1,3 +1,46 @@
 import updateTaskListOnDOM from '../markup-injectors/update-tasklist-dom-injection.js';
+import getParentElementId from '../utils/get-parent-element-id.js';
+import TaskManager from './task-manager-class.js';
 
-window.addEventListener('load', () => updateTaskListOnDOM());
+if (window.localStorage.length > 0) {
+  updateTaskListOnDOM('fullList');
+}
+
+document.querySelector('[data-refresh-btn]').addEventListener('click', () => {
+  window.location.reload();
```

Comment:
- [ ] **Optional:** Using Windows reload within your code isn't the best practice. This is because if you had async functions running, they would end without completion and could result in a lot of bugs.

Reply:
Hi @brytebee! and thank you for your review! 🙇🏼.

I am looking into other solutions for reloading, some of them include frameworks to reload certain parts of the page and the like.
I doubt this app will evolve any further, nevertheless, I thank you for your feedback! I will be useful for future projects involving more data, etc. I will make sure to stop using the window.location.reload().

Thank you @brytebee again! and happy coding partner! 👨🏼‍💻